### DISCUSSION TITLE
GitHub edu not working

### DISCUSSION THREAD

**Discussion by 27bchaos** [community/community#147178] [GitHub Education]
### Hi everyone,

Access to education.github.com was denied
You don't have authorization to view this page.
HTTP ERROR 403

**Comment by 27bchaos** [comment]
`Anyone` else having trouble??

**Reply by Kumala3** [reply]
@27bchaos It's working properly on my end. Which browser do you use and try to open it now. Let me know if you are still facing issues

**Reply by 27bchaos** [reply]
Im using chrome and still does not work.

**Reply by 27bchaos** [reply]
Its this link https://education.github.com/globalcampus/exchange

**Reply by 27bchaos** [reply]
@Kumala3

**Comment by 27bchaos** [comment]
Anyone?

**Reply by Kumala3** [reply]
It works properly on my end. Please try accessing that page using another browser.

**Comment by MohamadZeina** [comment]
Did you ever find an answer to this? I'm getting the same issue.
